docs/00-manuscripts/ieee/reviews/reviewer4.md

```diff
@@ -6,28 +6,24 @@ Although the general idea of the paper is sound, some major methodological and t
-We agree that the overall accuracies obtained across all models is not groundbreaking.
-However, we also believe that "success-bias" is a major issue in science which leads to rejection of studies due to low/fair accuracies of models.
+We agree that the predictive performances obtained are not overwhelming, and certainly lower than expected.
+However, we also believe that "success bias" is a major issue in science which leads to rejection of studies due to low/fair accuracies of models.
 We think that this manuscript might suffer from this potentially due to the reported accuracies which might lead to the feeling of an "unsuccessful" study.
 Model accuracies depend on many factors (data, algorithm, preprocessing) and are therefore naturally limited by these factors.
 We believe that a correct methodology is more important than the overall achieved accuracy.
 
-We also agree that the sampling time of the in-situ data might be problematic with respect to phenology.
-Therefore we discussed this point in section "Data quality".
-Nevertheless, this was the data which was available and which, in our view, is good enough to conduct the presented analysis.
+We also agree that the image acquisition date may not be ideal with respect to phenology. As so often, the actual date was not entirely controlled by phenological considerations, but it got delayed due to the timing of project funding and other practical constraints. Nevertheless, the results do show that the data is useful for the purpose for which it was acquired.
+We discuss this point in section "Data quality".
 
-Thank you suggestion to use segmentation.
+The use of segmentation techniques is also an interesting recommendation.
 We agree that applying a tree-crown segmentation first could potentially increase the accuracy of some models.
-However, the actual effect of such a preprocessing would be unclear and make the study more complex.
-In addition it would pull away focus from the actual main points of the study - the comparison of (ensemble) filter-based feature selection methods in combination with narrow-band remote sensing data.
-
-Last, we would like to point out that the main focus of this manuscript is about the comparison of filter methods - in a multi-algorithm setting based on hyperspectral data.
-It might not fulfill highest standards with respect to data quality from an applied forestry point of view.
+However, the actual effect of such a preprocessing would be unclear and make the study more complex, involving additional hyperparameters that control the behaviour of the segmentation algorithm.
+In addition it would draw the focus away from the actual main points of the study - the comparison of (ensemble) filter-based feature selection methods in combination with narrow-band remote sensing data. We therefore consider this an excellent recommendation for future work, and in particular for the application of the proposed techniques to more heterogeneous stands.
```

Comment:
```suggestion
In addition it would draw the focus away from the actual main points of the study - the comparison of (ensemble) filter-based feature selection methods in combination with narrow-band remote sensing data.
We therefore consider this an excellent recommendation for future work, and in particular for the application of the proposed techniques to more heterogeneous stands.
```